### ISSUE TITLE
Multiple Search terms crash

### ISSUE THREAD

**Issue by wisespace-io** [stephanrauh/ngx-extended-pdf-viewer#269]
I noticed that when we programmatically set multipleSearchTerms, are added extra '/n' to the query string, which reflects in extra empty lines in the search box, that can be noticed in https://github.com/stephanrauh/ngx-extended-pdf-viewer/blob/b398af9e6bc2466ae35a590cd73b30f2e75e876c/projects/ngx-extended-pdf-viewer/src/lib/ngx-extended-pdf-viewer.service.ts#L27

These extra carriage return are causing a crash If you remove from the findbox only the search terms and leave them (empty lines due extra '/n'), the preview will become unresponsive for while, that can be noticed in https://pdfviewer.net/find, but it does not crash. However in my application it becomes unresponsive and crashes later.

If I set "separated by word boundaries" options when I have  multipleSearchTerms it does not crash as this option ignores empty lines due the extras '/n'. However, this option is not available programmatically. Can you make this option available as well?

**Comment by stephanrauh** [comment]
I've fixed the crash. Thanks for pointing it out!

I've omitted the "findIndividualWords" option because I don't think it adds any additional value. Actually, I've even tried to implement it this evening, but it's more complicated than I thought. So I'd prefer not to implement it.

BTW, the implementation you've quoted feels like a quick hack indeed. But it does the trick, and every clean solution is much more verbose, so I'd like to leave it a that, too.
